gui/default/index.html

```diff
@@ -437,6 +437,12 @@ <h4 class="panel-title">
           </div>
         </div>
         <span class="pull-right">
+          <button type="button" class="btn btn-sm btn-default" ng-click="pauseAllFolders()">
+            <span class="fa fa-refresh"></span>&nbsp;<span translate>Pause All Folders</span>
+          </button>
+          <button type="button" class="btn btn-sm btn-default" ng-click="resumeAllFolders()">
+            <span class="fa fa-refresh"></span>&nbsp;<span translate>Resume All Folders</span>
+          </button>
```

Comment:
I wonder if we should hide the buttons if all folders are resumed or paused etc?